### PR TITLE
Update mysql_database resource query action.

### PR DESCRIPTION
**Pull request by jhou4**
# Description

Make mysql_database resource query action to work properly.

## Issues Resolved

https://github.com/sous-chefs/mysql/issues/678

## Check List

- [X] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
-## 11.0.3 - *2022-02-02*
+## 11.0.3 - *2022-02-09*
 
 - Remove delivery and move to calling RSpec directly via a reusable workflow
 - Update tested platforms
+- Update mysql_database.rb for query action
```
Comment (damacus) [inline CHANGELOG.md:11]: Please place the new changes under UNRELEASED, and our automation will sort the rest out.

**Review by damacus — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (damacus) [inline CHANGELOG.md:11]: Allow 
```suggestion
- Allow run_query to take a database name as an argument
```

**Comment by damacus** [comment]
@jhou4 thanks for the contribution 👏🏼 Couple of minor amendments and we can get this shipped shortly

**Review by damacus — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [11.0.4](https://github.com/sous-chefs/mysql/releases/tag/11.0.4)

**Pull request merged**